src/threads_and_async.md

```diff
@@ -30,11 +30,11 @@ Blocking our only thread with a heavy computation stops our application from pro
 There are multiple possibilities to address this common problem. In this chapter we'll have a look at Commands which is a simple yet extremely powerful mechanism which covers most use cases.
 
 Let's say you have an application that fetches data from a website.
-This leaves us in a similar situation as before: If we a synchronous HTTP library in the update function, we will block our main thread and freeze the application until the server responds.
+This leaves us in a similar situation as before: If we use a synchronous HTTP library in the update function, we will block our main thread and freeze the application until the server responds.
 So instead, we're going to use Commands in this example.
 
-Commands are background tasks that can be spawned using a `ComponentSender` or `FactoryComponentSender`.
-They run until they return their result as a `CommandOutput` message that will be processes by the component again.
+Commands are background tasks that can be spawned using a `ComponentSender` or a `FactoryComponentSender`.
+They run until they return their result as a `CommandOutput` which in turn is processed again by the component.
 
```

Comment:
That's more of an oversight from my side: `FactoryComponentSender` has been shortened to `FactorySender` which should also be updated in the book,